indispensable-josef/src/jvmTest/kotlin/at/asitplus/signum/indispensable/josef/JwkTest.kt

```diff
@@ -70,8 +69,8 @@ class JwkTest : FreeSpec({
             certificateSha256Thumbprint = Random.nextBytes(32),
         )
 
-        val serialized = joseCompliantSerializer.encodeToString(jwk)
-        val parsed = joseCompliantSerializer.decodeFromString<JsonWebKey>(serialized)
+        val serialized = jwk.serialize()
```

Comment:
Didn't we deprecate those `serialize()` methods in another PR? (also below)

Reply:
then i guess i messed up rebasing…